components/tree-select/demo/status.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'nz-demo-tree-select-status',
+  template: `
+    <nz-space nzDirection="vertical" style="width:100%;">
```

Comment:
Add `NzSpaceModule` to `demo/module` to make tests work as expected

Reply:
i forgot to submit